tools/setup_helpers/rocm_env.py

```diff
@@ -0,0 +1,59 @@
+import os
+from pathlib import Path
+
+
+def check_if_rocm() -> bool:
+    if os.path.exists("/opt/rocm"):
+        return True
+    if os.environ.get("ROCM_PATH") is not None:
+        return True
+    # Only used for Windows builds
```

Comment:
We can get rid of this comment, since `USE_ROCM` env var is usable even for Linux builds.